### PR TITLE
Enable plugin tests in CI for GHC 9.4

### PR DESCRIPTION
**Pull request by wz1000**
We neglected to enable testing in CI of all the newly patched plugins for GHC 9.4

### CODE REVIEW

**Review by pepeiborra — APPROVED** [review]
Thanks for noticing this!

**Comment by michaelpj** [comment]
Oof, good catch!

**Comment by mpickering** [comment]
Good work Zubin.

**Review by konn — APPROVED** [review]
Good catch, thanks! 👍

File: plugins/hls-refactor-plugin/test/Main.hs [hunk 1]
```diff
@@ -1669,8 +1669,10 @@ suggestImportTests = testGroup "suggest import actions"
     , test True []          "f = empty"                   []                "import Control.Applicative (empty)"
     , test True []          "f = empty"                   []                "import Control.Applicative"
     , test True []          "f = (&)"                     []                "import Data.Function ((&))"
-    , test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
-    , test True []          "f = Data.List.NonEmpty.nonEmpty" []            "import qualified Data.List.NonEmpty"
+    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name"
+      $ test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
+    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name"
+      $ test True []          "f = Data.List.NonEmpty.nonEmpty" []            "import qualified Data.List.NonEmpty"
```
Comment (fendor) [inline plugins/hls-refactor-plugin/test/Main.hs:1675]: This seems like rather important regressions, we ought to create an issue for that

File: plugins/hls-refactor-plugin/test/Main.hs [hunk 2]
```diff
@@ -1679,14 +1681,17 @@ suggestImportTests = testGroup "suggest import actions"
     , test True []          "f = (.|.)"                   []                "import Data.Bits (Bits((.|.)))"
     , test True []          "f = (.|.)"                   []                "import Data.Bits ((.|.))"
     , test True []          "f :: a ~~ b"                 []                "import Data.Type.Equality ((~~))"
-    , test True
+    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name"
+      $ test True
```
Comment (fendor) [inline plugins/hls-refactor-plugin/test/Main.hs:1685]: Same here and all the subsequent tests

**Review by fendor — APPROVED** [review]
LGTM

File: plugins/hls-refactor-plugin/test/Main.hs (hunk 1 above)
Comment (wz1000) [reply plugins/hls-refactor-plugin/test/Main.hs:1675]: I opened https://gitlab.haskell.org/ghc/ghc/-/issues/22130 a while ago but I didn't realise that it also affected import suggestions. Unfortunately it seems difficult to fix this in HLS and we have to wait for a fix in GHC.

Comment (wz1000) [reply plugins/hls-refactor-plugin/test/Main.hs:1675]: I've updated the message to point to this ticket.

**Pull request merged**